src/components/basic/Table.tsx

```diff
@@ -33,6 +33,15 @@ export const Table = View<Props>(({ headers, rows, onClickRow, ...props }) => {
 
 const Container = styled.table`
   width: 100%;
+  @media (max-width: 590px) {
+    font-size: 10px;
+    margin-left: 0px;
+    margin-right: 0px;
+    padding: 0;
+  }
+  @media (max-width: 400px) {
```

Comment:
can you define these values as UI constants with meaningful names?

Reply:
> can you define these values as UI constants with meaningful names?

Nice catch! updated it here https://github.com/kixlab/kuiz/pull/82/commits/a57bc9ea68f6c46b4a6fd4f4d306b6a7ef5b8941 :)